apps/ghostfolio/docker-compose.yml

```diff
@@ -0,0 +1,77 @@
+version: "3.9"
+
+services:
+  ghostfolio:
+    container_name: ghostfolio
+    image: ghostfolio/ghostfolio:1.300.0
+    restart: unless-stopped
+    ports:
+      - ${APP_PORT}:3333
+    environment:
+      NODE_ENV: production
+      HOST: 0.0.0.0
+      PORT: 3333
+      ACCESS_TOKEN_SALT: $GHOSTFOLIO_ACCESS_TOKEN_SALT
+      DATABASE_URL: postgresql://ghostfolio:${GHOSTFOLIO_DB_PASSWORD}@ghostfolio-postgres:5432/ghostfolio?sslmode=prefer
+      JWT_SECRET_KEY: ${GHOSTFOLIO_JWT_SECRET_KEY}
+      POSTGRES_DB: ghostfolio
+      POSTGRES_USER: ghostfolio
+      POSTGRES_PASSWORD: ${GHOSTFOLIO_DB_PASSWORD}
+      REDIS_HOST: ghostfolio-redis
+      REDIS_PASSWORD: ${GHOSTFOLIO_REDIS_PASSWORD}
+      REDIS_PORT: 6379
+    networks:
+      - tipi_main_network
+    depends_on:
```

Comment:
consider adding condition for depends_on instructions, so main app will not start until dependencies are ready to serve:
```
depends_on:
      ghostfolio-redis:
        condition: service_healthy
      ghostfolio-postgres:
        condition: service_healthy
```